task/hiffy/src/stm32g0.rs

```diff
@@ -18,49 +18,49 @@ pub struct Buffer(u8);
 pub enum Functions {
     Sleep(u16, u32),
     #[cfg(feature = "gpio")]
-    GpioInput(drv_stm32g0_sys_api::Port, drv_stm32g0_sys_api::GpioError),
+    GpioInput(drv_stm32xx_sys_api::Port, drv_stm32xx_sys_api::GpioError),
```

Comment:
This is a reminder that I really need to retool Humility to be using the IDL calls for `humliity gpio`!